app/assets/javascripts/index/tileinfo.js

```diff
@@ -0,0 +1,94 @@
+OSM.TileInfo = function(map) {
+  var tile;
+
+  function lon2x(lon, zoom) {
+    return Math.floor((lon + 180) / 360 * Math.pow(2, zoom));
+  }
+
+  function lat2y(lat, zoom) {
+    return Math.floor((1 - Math.log(Math.tan(lat * Math.PI / 180) + 1 /
+                                    Math.cos(lat * Math.PI / 180)) / Math.PI) / 2 * Math.pow(2, zoom));
+  }
+
+  function x2lon(x, z) {
+    return x / Math.pow(2, z) * 360 - 180;
+  }
+
+  function y2lat(y, z) {
+    var n = Math.PI - 2 * Math.PI * y / Math.pow(2, z);
+    return 180 / Math.PI * Math.atan(0.5 * (Math.exp(n) - Math.exp(-n)));
+  }
+
+  var page = {};
+
+  page.pushstate = page.popstate = function(path) {
+    OSM.loadSidebarContent(path, function () {
+      page.load(path, true);
+    });
+  };
+
+  page.load = function(path, noCentre) {
+    var params = querystring.parse(path.substring(path.indexOf('?') + 1)),
+        lat = parseFloat(params.lat),
+        lon = parseFloat(params.lon),
+        zoom = parseInt(params.zoom),
+        y = lat2y(lat, zoom),
```

Comment:
Yes, by multiplying by a tile size. See [this code](https://github.com/openstreetmap/openstreetmap-website/pull/1450/files#diff-7de9168b78d71bcdf442edf8b8aa44b9R72).